package.json

```diff
@@ -3,9 +3,12 @@
   "version": "6.6.1",
   "description": "super fast, all natural json logger",
   "main": "pino.js",
+  "type": "commonjs",
+  "types": "pino.d.ts",
   "browser": "./browser.js",
   "files": [
```

Comment:
@mcollina and @davidmarkclements I thought we removed this difficult to maintain property from the `package.json`?

Reply:
`fastify` uses `.npmignore` instead. If that is the way, I can use same approach here as well.

Reply:
I didn't remember we used this here even. Let's not change it in this PR.

Reply:
Agreed. Just asking because it got added in `pino-std-serializers` with this types stuff.

Reply:
@jsumners pino-std-serializer currently bundles all the garbage such as travis config, which felt not ideal. I can remove it from there if you feel strongly about it.

Reply:
Looks like we actually settled on `files` 🤦‍♂️ 

https://github.com/pinojs/pino/issues/136